iron-multi-selectable.html

```diff
@@ -97,6 +98,16 @@
       }
     },
 
+    // Create one array at a time to prevent stackoverflow when the fallback
+    // is not a valid selector. If you would do this in _selectMulti then a
+    // new array will be created every time due to a different scope.
+    _setFallbackArray: function(fallbackSolution) {
+      this._fallbackArray = [fallbackSolution];
+      if (this._shouldUpdateSelection) {
+        this._updateSelected();
+      }
```

Comment:
Maybe we test for if there are no selected items and update then too? Specifically, if you have a thing with IronSelectableBehavior, and it has no selected items and no `fallbackSelection`, should setting `fallbackSelection` then update the selection?